### ISSUE TITLE
Unloading Combines

### ISSUE THREAD

**Issue by Stu1986** [Stephan-S/FS22_AutoDrive#692]
Hi all
I have set AD to unload a combine into a trailer, it seems to be having an issue where the combine unloads about 5% tractor drives off parks and then is called back again then repeats the same, I have tried the combine pakred while unloading but still happens 🙄

**Comment by Iwan1803** [comment]
Some more informations can be usefull...
Is CP driving the combine and if yes: what are the settings for the combine in AD and CP?

**Comment by rohne83** [comment]
@Stu1986 can you please have a look at the videos of #656 and confirm whether you wanted to report the same behaviour? If yes, then that one issue should be enough for the same topic, and this one here could be closed. Otherwise you should provide the information requested by Iwan. Also screenshots could help.

**Comment by GTA3Fan** [comment]
I have experienced what they are referring to. 

It is very similar to what I posted except the combine does get the trigger to unload, however, it appears the tractor/trailer doesn't get the signal to stop while combine is unloading and just keeps on driving. So then combine clears the call because it unloaded just enough below threshold and continues to harvest but only makes it a few  feet before stopping and calling the trailer again.

I had this happen a few times and it looped maybe 4 times before I caught it. There were little piles of straw every 45ft or so. When I catch it again I'll record it with debug info.
